test/e2e/as-path-with-rewrites/as-path-with-rewrites.test.ts

```diff
@@ -0,0 +1,13 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('as-path-with-rewrites', () => {
+  const { next } = nextTestSetup({
+    files: __dirname,
+  })
+
+  it('should not include internal query params in `asPath` and `req.url`', async () => {
```

Comment:
Nit should we add this to an existing test suite so that we aren't creating a deployment for one assertion?